CHANGELOG.md

```diff
@@ -100,6 +100,48 @@ Alternate implementation which directly queries from middleware using view call
         .unwrap();
   ```
 
+* Added field `socket` to `OperatorInfo` and a method `get_operator_socket` to retrieve the socket from the `AvsRegistryServiceChainCaller` in PR [463](https://github.com/Layr-Labs/eigensdk-rs/pull/463).
```

Comment:
We should split this into two: "Added field `socket` to `OperatorInfo`" is a breaking change, while "Added a method `get_operator_socket` to retrieve the socket" goes in the "Added" section.